extension/shepherd/config.js

```diff
@@ -3,6 +3,7 @@
  */
 export const config = {
     apiUri: "http://localtest.me:8080/api/",
+    websiteUri: "http://localtest.me:4200/",
```

Comment:
This should be retrieved from the current URI

Reply:
This is the Shepherd website's page but I agree the variable's name is not clear